packages/cli/plopfile.js

```diff
@@ -0,0 +1,46 @@
+const promptDirectory = require('inquirer-directory');
+
+module.exports = function(plop) {
+  plop.setPrompt('directory', promptDirectory);
+
+  plop.setGenerator('ngxs', {
+    description: 'Create new store',
+    prompts: [
+      {
+        type: 'input',
+        name: 'name',
+        message: 'Write store name, please...'
```

Comment:
How about just `Store name:` the `...` makes me think I'm waiting on something.

Reply:
done